tests/test_suites/llm/sft-nanov3-30BA3B-2n4g-fsdp2-lora.sh

```
@@ -0,0 +1,43 @@
+#!/bin/bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
+source $SCRIPT_DIR/common.env
+
+# ===== BEGIN CONFIG =====
+NUM_NODES=2
+GPUS_PER_NODE=4
+STEPS_PER_RUN=20  # step_time ~ 10sec
+MAX_STEPS=20
+NUM_RUNS=$(( (MAX_STEPS + STEPS_PER_RUN - 1) / STEPS_PER_RUN ))  # Round up
+NUM_MINUTES=60 # Usually 15 minutes is enough for 20 steps, but we add a buffer of 3 minutes in metrics check
```

Shellcheck (0.11.0)

[warning] NUM_MINUTES appears unused. Verify use (or export if used externally).

(SC2034)